Makefile.target

```diff
@@ -135,7 +135,7 @@ obj-y += memory.o savevm.o cputlb.o
 obj-y += memory_mapping.o
 obj-y += dump.o
 obj-$(call lnot,$(CONFIG_ROCKER)) += hw/net/rocker/qmp-norocker.o
-LIBS+=$(libs_softmmu)
+LIBS+=$(libs_softmmu) -lJudy
```

Comment:
libjudy is used in the P4 behavioral model. That might be a big change for
us to migrate to something else like GHash.
Do you think it can result in "no-go"?

On Sun, Apr 19, 2015 at 9:55 PM, Scott Feldman <notifications@github.com>
wrote:

> In Makefile.target
> <https://github.com/pbhide/qemu-rocker/pull/1#discussion_r28664124>:
>
> > @@ -135,7 +135,7 @@ obj-y += memory.o savevm.o cputlb.o
> >  obj-y += memory_mapping.o
> >  obj-y += dump.o
> >  obj-$(call lnot,$(CONFIG_ROCKER)) += hw/net/rocker/qmp-norocker.o
> > -LIBS+=$(libs_softmmu)
> > +LIBS+=$(libs_softmmu) -lJudy
>
> Is there something already like libjudy in the qemu build tree? libjudy
> looks nice but I can see a road-block trying to get it included for
> upstream qemu.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pbhide/qemu-rocker/pull/1/files#r28664124>.
>